index.html

```diff
@@ -486,10 +415,17 @@ <h6 class="code-block-title">Code Block</h6>
       </div>
     </footer>
 
+    <!-- Load AJS -->
+    <script src="./segment.js"></script>
+
+    <script src="https://cdn.jsdelivr.net/gh/highlightjs/cdn-release@11.8.0/build/highlight.min.js"></script>
+
     <!-- Bootstrap JS -->
-    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/js/bootstrap.bundle.min.js"></script>
```

Comment:
you were importing bootstrap.js twice, once at the top and once at the bottom. this was causing some bugs, like the "See Details" not toggling correctly